adoc/chapters/programming_interface.adoc

```diff
@@ -16050,6 +16066,19 @@ info::kernel_device_specific::compile_sub_group_size
 
 |====
 
+[NOTE]
+====
+The _guaranteed_ minimum amount of local memory available for dynamic allocation
+through [code]#local_accessor# can be calculated by subtracting the value
+returned for [code]#info::kernel_device_specific::local_static_mem_size# from
+the value returned for [code]#info::device::local_mem_size# (see
+<<sec:device-class>>). 
+
+Backends may support reuse of local memory for multiple allocations, resulting
+in more local memory being available for dynamic allocation through
+[code]#local_accessor#, but are not required to do so and relying on this 
+behavior may result in non-portable code.
```

Comment:
It is unclear to me what you are thinking to.
Would this optimization not have any effect on the value returned by your introspection function anyway?